### PR TITLE
fix mixed content errors when serving over https

### PR DESCRIPTION
**Pull request by shiltemann**
When I tried to serve the site from my own fork, I got mixed content errors when serving over https, which caused the slides not to render for instance, this fixed that for me :)

### CODE REVIEW

**Comment by shiltemann** [comment]
hmm, still doesnt seem to work on firefox btw.. will look into this further

(xref: https://github.com/gnab/remark/issues/340)

**Comment by shiltemann** [comment]
ok, nevermind, it was just my script blocker plugin in firefox, this does work :)

**Comment by shiltemann** [comment]
ok, changed the base url in jekyll config as well, don't know if you need to change anything in the repo settings (it wouldnt even let me set my gh-pages to use plain http, but seems like we do here? )

File: _layouts/home.html [hunk 1]
```diff
@@ -100,7 +100,7 @@
     <a href="https://www.uni-freiburg.de/"><img src="https://raw.githubusercontent.com/bgruening/presentations/a2e38e4b007994af798320db3a0131c4bb891c0e/shared/resources/img/logo_freiburg.jpg" width="15%" style="max-width:100%;"></a>
     <br>
     <a href="https://wiki.galaxyproject.org/Teach/GTN"><img src="./shared/images/GTNLogo1000.png" width="15%" style="max-width:100%;"></a>
-    <a href="http://www.france-bioinformatique.fr/"><img src="http://www.france-bioinformatique.fr/sites/default/files/ifb-logo_1.png" width="15%" style="max-width:100%;"></a>
+    <a href="http://www.france-bioinformatique.fr/"><img src="https://www.france-bioinformatique.fr/sites/default/files/ifb-logo_1.png" width="15%" style="max-width:100%;"></a>
```
Comment (nsoranzo) [inline _layouts/home.html]: You can also change the link target to https

**Comment by bgruening** [comment]
LGTM! Thanks!

**Pull request merged**